go/weed/weed_server/filer_server_handlers.go

```diff
@@ -68,11 +79,20 @@ func (fs *FilerServer) GetOrHeadHandler(w http.ResponseWriter, r *http.Request,
 		fs.listDirectoryHandler(w, r)
 		return
 	}
-	fileId, err := fs.filer.FindFile(r.URL.Path)
-	if err == leveldb.ErrNotFound {
-		glog.V(3).Infoln("Not found in db", r.URL.Path)
-		w.WriteHeader(http.StatusNotFound)
-		return
+	var fileId string
+	uri_elements := strings.Split(r.URL.Path, "/")
+	//glog.V(1).Infof("debug uri_elements %#v", uri_elements)
+	if len(uri_elements) == 2 {
```

Comment:
It's better not to have this feature. This should be taken care by the volume server or master server redirecting. Mix it to file server is not ideal.